lib/ansible/runner/credential/rhv_credential.rb

```diff
@@ -0,0 +1,52 @@
+module Ansible
+  class Runner
+    class RhvCredential < Credential
+      def self.auth_type
+        "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::RhvCredential"
+      end
+
+      # Modeled off of rhv injectors for awx:
+      #
+      #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/models/credential/__init__.py#L1035-L1054
+      #
+      def env_vars
+        {
+          "OVIRT_INI_PATH" => ovirt_ini_file,
+          "OVIRT_URL"      => auth.host || "",
+          "OVIRT_USERNAME" => auth.userid || "",
+          "OVIRT_PASSWORD" => auth.password || "",
+        }
+      end
+
+      def write_config_files
+        write_ovirt_ini_file
+      end
+
+      private
+
+      def write_ovirt_ini_file
+        ovirt_data = %W[
+          [ovirt]
+          ovirt_url=#{auth.host}
+          ovirt_username=#{auth.userid}
+          ovirt_password=#{auth.password}
+        ]
+
+        # TODO:  NickL:  Should we support this?
+        #
+        # awx ref:
+        #
+        #   https://github.com/ansible/awx/blob/1242ee2b/awx/main/models/credential/__init__.py#L1046
+        #
+        # ovirt_data << "ovirt_ca_file=#{auth.auth_key}" if auth.auth_key
+
+        File.write(ovirt_ini_file, ovirt_data.join("\n"))
+        File.chmod(0o0600, ovirt_ini_file)
+      end
+
+      def ovirt_ini_file
+        File.join(base_dir, "ovirt.ini")
```

Comment:
Quick note here:  I changed this from `rhv_credentials` (what it was in the previous version) since this is what the file is normally called:

https://github.com/openshift/openshift-ansible-contrib/blob/d16d2f73/reference-architecture/rhv-ansible/inventory/ovirt.ini.example#L21-L34